run-poc/process-word-pairs.sh

```diff
@@ -36,11 +36,11 @@ case $1 in
 esac
 
 # Punt if the cogserver has crashed: use netcat to ping it.
-haveping=`echo foo | nc -N localhost $PORT`
+haveping=`echo foo | nc localhost $PORT`
 if [[ $? -ne 0 ]] ; then
 	exit 1
 fi
 
 # Submit instruction to the cogserver
-echo -e "(load \"$module\")" | nc -N localhost $PORT
-echo -e "($func \"$cnt_mode\")" | nc -N localhost $PORT
+echo -e "(load \"$module\")" | nc localhost $PORT
+echo -e "($func \"$cnt_mode\")" | nc localhost $PORT
```

Comment:
I assume that you have lots of experience running this without the `-N` flag.  I don't quite remember why I added the `-N` flag, but I think the cogserver was running out of sockets, resulting in errors and/or causing it to go slow.  You may want to double-check -- the cogserver might run faster with the `-N` flag.

Reply:
the "-N" flag is required by some versions of netcat but not others. You're right, I should keep the -N and only remove it on systems that don't understand it (there's a note about this in the readme).